scripts/ingest-tarball.sh

```diff
@@ -258,7 +258,12 @@ version=$(echo "${tar_file_basename}" | cut -d- -f2)
 contents_type_dir=$(echo "${tar_file_basename}" | cut -d- -f3)
 tar_first_file=$(tar tf "${tar_file}" | head -n 1)
 tar_top_level_dir=$(echo "${tar_first_file}" | cut -d/ -f1)
-tar_contents_type_dir=$(tar tf "${tar_file}" | head -n 2 | tail -n 1 | cut -d/ -f2)
+# Handle longer prefix with project name in dev.eessi.io
+if [ "${cvmfs_repo}" = "dev.eessi.io" ]; then
+    tar_contents_type_dir=$(tar tf "${tar_file}" | head -n 2 | tail -n 1 | cut -d/ -f3)
+else
+    tar_contents_type_dir=$(tar tf "${tar_file}" | head -n 2 | tail -n 1 | cut -d/ -f2)
+fi
```

Comment:
```suggestion
# Handle longer prefix with project name in dev.eessi.io
if [ "${cvmfs_repo}" = "dev.eessi.io" ]; then
    tar_contents_start_level=3
else
    tar_contents_start_level=2
fi
tar_contents_type_dir=$(tar tf "${tar_file}" | head -n 2 | tail -n 1 | cut -d/ -f${tar_contents_start_level})
```

Something like this (and using that same variable in `check_arch` and `check_os`) seems to work.

Reply:
Ah, this is much better than what I pushed just now in 857a339

Reply:
I'll revert my changes